Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+pipeline {
+  agent none
+    environment {
+      JENKINS_RUN = '1'
+    }
+  stages {
```

Comment:
Cool feature! I mean pipelines in general.
What we probably want is execute tests (coverage, valgring, deb-unstable etc.) in parallel and for that as far as I could understand we need one mega-stage with multiple parallel steps (see https://jenkins.io/doc/book/pipeline/jenkinsfile/#parallel-execution). It can be fixed later though.

Reply:
Yes, I'm currently trying to run the debian install test in parallel in the branch `test/jenkins-debinstall`. It should be close to be working now.